plugin/restore/mysql/mysql.go

```diff
@@ -132,13 +133,13 @@ func (r *Restore) DeleteOldDatabase(ctx context.Context, database string, suffix
 	return nil
 }
 
-// getPITRDatabaseName composes a pitr database name
+// Composes a pitr database name that we use as the target database for full backup recovery and binlog recovery.
```

Comment:
TBH,  I still couldn't comprehend the relationship/difference between getPITRDatabaseName and getPITROldDatabaseName. Need some ascii comment e.g.

table -> pitrtable_xxx
pitrable_xxx -> pitrable_xxx_old
pitrable_xxx_old - > table

Reply:
Indeed. BTW it's at the database level, so the table names are preserved.

The ASCII comments may be:
```
database -> database_pitr_1653018005
database -> database_pitr_1653018005_old
```

WDYT? @tianzhou

Reply:
dbfoo -> dbfoo_pitr_1653018005
dbfoo -> dbfoo_pitr_1653018005_old

And this only conveys one direction, but we are doing swap here, so we also need to comment the other way around.